src/parser/core/modules/__tests__/PrecastAction.ts

```diff
@@ -0,0 +1,122 @@
+import PrecastAction from 'parser/core/modules/PrecastAction'
```

Comment:
nit; per other pr
```suggestion
import PrecastAction from '../PrecastAction'
```

That said, keep the imports below as-is - they're not the thing being tested